style.css

```diff
@@ -0,0 +1,57 @@
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+a{
+    color: #e8eaed;
+    text-decoration: none;
+}
+iframe{
+    position: absolute;
+    margin-left: -10px;
+    box-shadow: 3px 3px 3px black;
+}
+
+#info{
+    position: absolute;
+    margin: 430px 500px 300px 30px;
+}
+
+body{
+    background-color: #282828;
+}
+
```

Comment:
Kindly add the body style at the top of the page. This allows for a better code readability.